SportsNews/Helpers/UserHelper.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using System.Linq;
+using System.Security.Claims;
+
+namespace SportsNews
+{
+    public static class UserHelper
+    {
+        public static HtmlString CreateUserFullName(this IHtmlHelper html, ClaimsPrincipal user)
```

Comment:
This method is not creating User Full Name. 
It's getting User Full Name from Claims. 
Change naming to GetUserFullNameFromClaims